Add isExactMatchRequired property (#19117)
microsoftgraph/microsoft-graph-docs: concepts/connecting-external-content-manage-schema.md

*Refine results by `tags`, a refinable property.*

> ### Exact Match Required

Reviewer: ```suggestion
### Exact match required
```

> If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can specify exact matching.

Reviewer: Line 78, second sentence: It sound like the property is the one that specifies. Could you please rephrase to indicate what is the desired action from the developer? How can the developer specify this? 

Sugestion:
```suggestion
If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can by specified for exact matching.
```

Reviewer 2: There is a typo in this revision: 
If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can >>by<< specified for exact matching.
Should be: If exact matching is specified for a property, then the full string value will be indexed. Only non-searchable properties can be specified for exact matching.

Reviewer: Great catch @dodaromike. Fixed. Thank you!

For example, the `ticketId` property is both queryable and specifies exact matching.

> - Querying "ticketId:CTS-ce913b61" will return the item with a ticket ID property "CTS-ce913b61"
> - Querying "ticketId:CTS" will NOT return the item with ticket ID "CTS-ce913b61"

Reviewer: ```suggestion
- Querying "ticketId:CTS-ce913b61" will return the item with a ticket ID property "CTS-ce913b61".
- Querying "ticketId:CTS" will NOT return the item with ticket ID "CTS-ce913b61".
```

Similarly, the `tags` property also specifies exact matching.

> - Querying "tags:contoso" will return any item with the tag "contoso"
> - Querying "tags:contoso" will NOT return items with the tag "contoso ticket"

Reviewer: ```suggestion
- Querying "tags:contoso" will return any item with the tag "contoso".
- Querying "tags:contoso" will NOT return items with the tag "contoso ticket".
```

> For example, there may be a scenario where the item property is a GUID-formatted string. If this property must be matched exactly for item queries, then please specify that `isExactMatchRequired` is true.

Reviewer: In general, we avoid using the word "please" unless we are requesting the user to perform an inconvenient action.
Details here: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please

```suggestion
For example, there may be a scenario where the item property is a GUID-formatted string. If this property must be matched exactly for item queries, specify that `isExactMatchRequired` is true.
```

> The `title` property does not specify exact matching. If nothing is specified, then the `isExactMatchRequired` is false. The `title` property will be tokenized based on the item content language's tokenization rules.

Reviewer: Suggestion:
```suggestion
The `title` property does not specify exact matching. If nothing is specified, then `isExactMatchRequired` is false. The `title` property will be tokenized based on the tokenization rules of the language of the item content.
```

> - Querying "title:Contoso Title" will return any item containing "Contoso" or "Title" in the `title` property

Reviewer: ```suggestion
- Querying "title:Contoso Title" will return any item containing "Contoso" or "Title" in the `title` property.
```
